src/libs/auth.js

```diff
@@ -185,9 +186,12 @@ authStore.subscribe(withErrorReporting('Error checking TOS', async (state, oldSt
 
 authStore.subscribe(async (state, oldState) => {
   if (!oldState.acceptedTos && state.acceptedTos) {
-    window.Appcues && window.Appcues.identify(state.user.id, {
-      dateJoined: parseJSON((await Ajax().User.firstTimestamp()).timestamp).getTime()
-    })
+    if (window.Appcues) {
+      window.Appcues.identify(state.user.id, {
+        dateJoined: parseJSON((await Ajax().User.firstTimestamp()).timestamp).getTime()
+      })
+      window.Appcues.on('all', captureAppcuesEvent)
```

Comment:
I don't see anything in the Appcues docs (https://docs.appcues.com/article/302-working-with-client-side-appcues-events) that say the callback can return a `Promise` and they'll make sure it completes (if that's even a thing) but I also can't find any way I can resolve the promise myself that doesn't just result in a new promise... and it seems to work, so... I think this is fine? 🤷

Reply:
It's not a thing to make sure it completes; basically, they don't care if the callback returns a promise, or indeed if it returns anything at all. They just call it.

```js
const aPromise = async () => {
  await doSomething()
}

aPromise()
```

They just dispatch and forget it, so we're good.